### PR TITLE
Remove -E all from installation instructions

### PR DESCRIPTION
**Pull request by seamusp**
Using poetry install -E all no longer works since [tool.poetry.extras] was removed from pyproject.toml, so I removed it and the blurb explaining it from the installation instructions.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -80,10 +80,10 @@ For example, to contribute to `langchain` run `cd libs/langchain` before getting
 To install requirements:
 
 ```bash
-poetry install -E all
+poetry install
```
Comment (eyurtsev) [inline .github/CONTRIBUTING.md:83]: Thanks! we definitely don't want `all`, we do want contributors to have the test group installed to run linters and local unit tests

```suggestion
poetry install --with test
```

**Review by eyurtsev — APPROVED** [review]
Thanks!

**Comment by baskaryan** [comment]
what do we mean by tool.poetry.extras was removed? im still seeing it in libs/langchain/pyproject.toml

**Comment by seamusp** [comment]
> what do we mean by tool.poetry.extras was removed? im still seeing it in libs/langchain/pyproject.toml

When I try running `poetry install -E all` it fails because tool.poetry.extras doesn't exist in the pyproject.toml in the root directory. What would you suggest the documentation say instead?

**Comment by eyurtsev** [comment]
@seamusp the directory structure changed a bit, if you're looking to contribute to the core langchain package, use the pyproject toml located: `libs/langchain/` instead of the one at the root level. That one does have `all` extra, though I still think that that's not a good default for contributors

**Pull request merged**